### PR TITLE
Remove versions on Emulator references and unused links from README

### PR DESCRIPTION
**Pull request by Zerryth**
Regards to requests by @cleemullins in PR https://github.com/Microsoft/BotBuilder-Samples/pull/999

## Proposed Changes
  - removed "v4" in references to Bot Framework Emulator
  - removed extra, unused links from the bottom of Echo README

### CODE REVIEW

File: generators/vsix-vs-win/BotBuilderVSIX-V4/ProjectTemplates/Multi-project/EchoBot/README.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ This bot has been created using [Microsoft Bot Framework][1], it shows how to cr
 * Type `dotnet run`.
 
 ## Interacting With Your Bot Using the Emulator
-Launch the [Microsoft Bot Framework Emulator v4][3] and open the generated project's `.bot` file.
+Launch the [Microsoft Bot Framework Emulator v4][3] and open the `$safeprojectname$.bot` file.
 
-# Testing the bot using Bot Framework Emulator **v4**
```
Comment (sgellock) [inline generators/vsix-vs-win/BotBuilderVSIX-V4/ProjectTemplates/Multi-project/EchoBot/README.md]: There are both version 3.x and 4.x releases of the Emulator.  I've been very prescriptive in calling out that it needs to be v4.  To make matters worse, the last release that went out was a v3 release so when you nav to the releases page, the first thing you will see is a v3 release.  I'd leave the call to action for v4 in the readme.

File: generators/vsix-vs-win/BotBuilderVSIX-V4/ProjectTemplates/Multi-project/EchoBot/README.md [hunk 2]
```diff
@@ -14,14 +14,14 @@ This bot has been created using [Microsoft Bot Framework][1], it shows how to cr
...
+# Testing the bot using Bot Framework Emulator
 [Microsoft Bot Framework Emulator][3] is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
 
-* Install the Bot Framework Emulator version 4.1.0 or greater from [here][4].
```
Comment (sgellock) [inline generators/vsix-vs-win/BotBuilderVSIX-V4/ProjectTemplates/Multi-project/EchoBot/README.md]: See comment above about v3/v4 confusion

File: generators/vsix-vs-win/BotBuilderVSIX-V4/ProjectTemplates/Multi-project/EchoBot/README.md [hunk 3]
```diff
@@ -14,14 +14,14 @@ This bot has been created using [Microsoft Bot Framework][1], it shows how to cr
...
+* Install the Bot Framework Emulator. (Download [here][4].)
 
-## Connect to the bot using Bot Framework Emulator **v4**
```
Comment (sgellock) [inline generators/vsix-vs-win/BotBuilderVSIX-V4/ProjectTemplates/Multi-project/EchoBot/README.md]: See comment above about v3/v4 confusion

**Comment by Zerryth** [comment]
See PR https://github.com/Microsoft/BotBuilder-Samples/pull/1014

**Pull request closed without merging**